modules/Book.js

```diff
@@ -0,0 +1,70 @@
+import printErrorMsg from './errorMsg.js';
+
+let books = JSON.parse(localStorage.getItem('books'));
+
+export default class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
+
+  addBook() {
+    const { id, title, author } = this;
+    const bookObj = { id, title, author };
+    books = JSON.parse(localStorage.getItem('books'));
+    if (title === '' || author === '') {
+      printErrorMsg('Please fill in all the fields');
+    } else if (books !== null) {
+      books.push(bookObj);
+      localStorage.setItem('books', JSON.stringify(books));
+      books = JSON.parse(localStorage.getItem('books'));
+      document.getElementById('title').value = '';
+      document.getElementById('author').value = '';
+    } else {
+      books = [];
+      books.push(bookObj);
+      localStorage.setItem('books', JSON.stringify(books));
+      books = JSON.parse(localStorage.getItem('books'));
+      document.getElementById('title').value = '';
+      document.getElementById('author').value = '';
+    }
+  }
+
+  removeBook() {
+    const { id } = this;
+    books = books.filter((book) => {
+      if (book.id !== id) {
+        return true;
+      }
+      return false;
+    });
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
+
+export function displayBook(id, title, author) {
```

Comment:
- [ ] Kindly refactor to arrow function.

Reply:
Resolved. Thanks